packages/subgraph/ponder.schema.ts

```diff
@@ -26,6 +42,24 @@ export const swap = onchainTable('swaps', (t) => ({
     createdAt: t.bigint(),
 }))
 
+// Denormalized events table for all analytics
+export const analyticsEvent = onchainTable('analytics_events', (t) => ({
+    id: t.text().primaryKey(), // ${txHash}-${logIndex}
+    spaceId: t.hex(),
+    eventType: t.text(), // 'swap', 'tip', etc.
```

Comment:
I believe we can use an enum here see https://ponder.sh/docs/schema/tables#enums

```
  import { onchainTable, onchainEnum, relations } from 'ponder'

  export const analyticsEventType = onchainEnum('analytics_event_type', [
      'swap',
      'tip',
      'join'
  ])

  eventType: analyticsEventType().notNull(),
```